content/applications/finance/accounting/fiscal_localizations/localizations/colombia.rst

```diff
@@ -1,276 +1,41 @@
-- **Factura de Importación**: This should be selected for importation
-  transactions.
-- **Factura de contingencia**: This is an exceptional type that is
-  used as a manual backup in case that the company is not able to use
-  the ERP and it's necessary to generate the invoice manually, when
-  this invoice is added to the ERP, this invoice type should be
-  selected.
-
-
-Invoice validation
-~~~~~~~~~~~~~~~~~~
-
-After the invoice is validated an XML file is created and sent
-automatically to Carvajal, this file is displayed in the chatter.
-
-.. image:: colombia/colombia13.png
-   :align: center
-
-An extra field is now displayed in "Other Info" tab with the name of
-the XML file. Additionally there is a second extra field that is
-displayed with the Electronic Invoice status, with the initial value
-"In progress":
-
-.. image:: colombia/colombia14.png
-   :align: center
-
-
-Reception of legal XML and PDF
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The electronic invoice vendor receives the XML file and proceeds to
-validate the structure and the information in it, if everything is
-correct the invoice status changes to "Validated" after using the
-"Check Carvajal Status" button in the Action dropdown. They then
-proceed to generate a Legal XML which includes a digital signature and
-a unique code (CUFE), a PDF invoice that includes a QR code and the
-CUFE is also generated.
-
-After this:
-
-- A ZIP containing the legal XML and the PDF is downloaded and
-  displayed in the invoice chatter:
-
-  .. image:: colombia/colombia15.png
-
-  .. image:: colombia/colombia16.png
-
-- The Electronic Invoice status changes to "Accepted"
-
-
-Common errors
-~~~~~~~~~~~~~
-
-During the XML validation the most common errors are usually related
-to missing master data. In such cases, error messages are shown in the
-chatter after updating the electronic invoice status.
+Electronic invoicing for Colombia is available for Odoo 12 and above, and requires the following
+modules:
 
-.. image:: colombia/colombia17.png
-   :align: center
+.. list-table::
+   :header-rows: 1
 
-After the master data is corrected, it's possible to reprocess the XML
-with the new data and send the updated version, using the following
-button:
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Colombia - Accounting`
+     - ``l10n_co``
+     - Default :doc:`fiscal localization package <../overview/fiscal_localization_packages>`
+   * - :guilabel:`Colombia E-invoice Integration`
+     - ``l10n_co_edi``
+     - :ref:`Carvajal e-invoicing integration <colombia/e-invoicing>`
 
-.. image:: colombia/colombia18.png
-   :align: center
+.. toctree::
+   :titlesonly:
 
-.. image:: colombia/colombia19.png
-   :align: center
+   colombia/configuration
+   colombia/workflows
+   colombia/reports
 
+Video tutorials
+===============
 
-Additional use cases
---------------------
+View the video(s) below for a general description of the localization, and how to configure it.
```

Comment:
@samueljlieber this playlist has been improved a lot since the last time we add the reference to te documentation.  We can replace it by something like this: 

Check the playlist below for a serie of smart classes that cover how to start from scatch, configiration, main workflow as well as special use cases. These videos can help you to achive a succesful implementation. 

cc. @vbe-odoo @fvz-odoo